doc/tutorials/core/random_generator_and_text/random_generator_and_text.markdown

```diff
@@ -95,7 +95,7 @@ Explanation
 
        line( image, pt1, pt2, randomColor(rng), rng.uniform(1, 10), 8 );
        imshow( window_name, image );
-       if( waitKey( DELAY ) >= 0 )
+       if( (waitKey( DELAY ) & 0xFF) >= 0 )
```

Comment:
Hmmm in fact the 0xFF here is not needed, simply because it has to be `any` key. Will remove the 0xFF in these cases to not make the code more complex then needed. Basically waitChar should be used if 0xFF is needed, else the 0xFF can go!